providers/certification-client/units/client-cert-odm-server-20-04.pxu

```diff
@@ -45,6 +45,7 @@ nested_part:
     after-suspend-bluetooth-manual
     after-suspend-ethernet-manual
     after-suspend-wwan-manual
+    acpi_oem_osi_strings
```

Comment:
If you implement a `acpi-manual` test plan, you can put it here instead of `acpi_oem_osi_strings` (
it's better to put it higher in the list, maybe above `audio-manual`):

```suggestion
    acpi-manual
```


If you don't, then you can just remove `acpi_oem_osi_strings`.